src/main/java/iitema/gypsypokemon/elements/blocks/Door.java

```diff
@@ -29,55 +29,7 @@ public Door(Direction dir) {
      */
     @Override
     protected boolean solid(Direction dir) {
-        if (this.openSides.get(dir)) {
-            if (this.item == null) {
-                return true;
-            }
-            return this.item.solid(dir);
-        }
-        return false;
-    }
-
-    /**
-     * If an item is on this field, return reference to it
-     *
-     * @param dir the direction the player is facing
-     * @return an item on the field or null if none
-     */
-    @Override
-    public ItemInterface getItem(Direction dir) {
```

Comment:
By removing this, you are allowing a player to get an item from the wrong side of the door,